### PR TITLE
feat: P4ADEV-512 add mypay datasource

### PR DESCRIPTION
**Pull request by antocalo**
#### How Has This Been Tested?

- Pre-Deploy Test
    - [ ] Unit
    - [ ] Integration (Narrow)
- Post-Deploy Test
    - [ ] Isolated Microservice
    - [ ] Broader Integration
    - [ ] Acceptance
    - [ ] Performance & Load

#### Types of changes

- [ ] PATCH - Bug fix (backwards compatible bug fixes)
- [ ] MINOR - New feature (add functionality in a backwards compatible manner)
- [x] MAJOR - Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/gov/pagopa/payhub/auth/mypay/config/JdbcTemplateConfig.java
Comment (and-mora) [inline src/main/java/it/gov/pagopa/payhub/auth/mypay/config/JdbcTemplateConfig.java:1]: is `JdbcTemplate` needed if we are going to use the Spring Data Repository class?

Comment (antocalo) [reply src/main/java/it/gov/pagopa/payhub/auth/mypay/config/JdbcTemplateConfig.java:1]: I removed JdbcTemplate, if we need we are going to add later

**Review by and-mora — APPROVED** [review]

**Comment by and-mora** [comment]
please try to fix the sonar issue or open a new tech debt task

**Comment by antocalo** [comment]
I'll try to fix with the next commit

> please try to fix the sonar issue or open a new tech debt task

**Comment by and-mora** [comment]
for the next times: this PR provides no value for the end user therefore the semantic tag `feat` is not really appropriate. Try `chore` for the PR that do not provide value nor fix any bugs.

**Pull request merged**